GDWeave/Script/ModUtil.cs

```diff
@@ -10,6 +10,58 @@ public interface IWaiter {
     public bool Check(Token token);
 }
 
+public class FunctionWaiter : IWaiter {
+    private MultiTokenWaiter _FunctionWaiter = new([
+        t => t.Type == TokenType.PrFunction,
+        t => t.Type == TokenType.Identifier,
+    ]);
+
+    public bool Matched {get; private set;} = false;
+    public bool Ready {get; private set;} = true;
+
+    private readonly string name;
```

Comment:
This should probably take a `waitForReady = false` like the other waiters do.

Reply:
Done